awscli/testutils.py

```diff
@@ -99,7 +99,7 @@ def get_aws_cmd():
     import awscli
     if AWS_CMD is None:
         # Try <repo>/bin/aws
-        repo_root = os.path.dirname(os.path.abspath(awscli.__file__))
+        repo_root = os.path.dirname(os.path.dirname(os.path.abspath(awscli.__file__)))
```

Comment:
Why did this need to change?

Reply:
don't need it! was messing around with a few things and left it in accidentally. will remove